testing/MLDB-174-regression.py

```diff
@@ -0,0 +1,224 @@
+        ds = mldb.create_dataset({ "id": "test", "type": "sparse.mutable" })
+        for row, x, y in ( ("ex1", 0, 0), ("ex2", 1, 1), ("ex3", 2, 2), ("ex4", 3, 3)):
+            ds.record_row(row, [ [ "x", x, 0 ], ["y", y, 0] ])
+        ds.commit()
+
+        self.glz_conf =  {
+            "glz": {
+                "type": "glz",
+                "verbosity": 3,
+                "normalize": False,
+                "link_function": 'linear',
+                "ridge_regression": False
+            }
+        }
+        
+        rez = mldb.put('/v1/datasets/wine_red', {
+            "type": "text.csv.tabular",
+            "params": {
+                "dataFileUrl": "https://s3.amazonaws.com/public.mldb.ai/datasets/wine_quality/winequality-red.csv",
+                "delimiter": ";"
+            }
+        })
+        
+        rez = mldb.put('/v1/datasets/wine_white', {
+            "type": "text.csv.tabular",
+            "params": {
+                "dataFileUrl": "https://s3.amazonaws.com/public.mldb.ai/datasets/wine_quality/winequality-white.csv",
+                "delimiter": ";"
+            }
+        })
+
+
+    def test_select_simple_regression(self):
+        modelFileUrl = "file://tmp/MLDB-174.cls"
+
+        trainClassifierProcedureConfig = {
+            "type": "classifier.train",
+            "params": {
+                "trainingData": { 
+                    "select": "{x} as features, y as label",
+                    "from": { "id": "test" }
+                },
+                "configuration": self.glz_conf,
+                "algorithm": "glz",
+                "modelFileUrl": modelFileUrl,
+                "equalizationFactor": 0.0,
+                "mode": "regression",
+                "runOnCreation": True
+            }
+        }
+
+        procedureOutput = mldb.put("/v1/procedures/cls_train", trainClassifierProcedureConfig)
+        mldb.log(procedureOutput.json())
+
+
+        functionConfig = {
+            "type": "classifier",
+            "params": {
+                "modelFileUrl": modelFileUrl
+            }
+        }
+
+        createFunctionOutput = mldb.put("/v1/functions/regressor", functionConfig)
+        mldb.log(createFunctionOutput.json())
+
+        selected = mldb.get("/v1/functions/regressor/application", input = { "features": { "x":10 }})
+
+        jsSelected = selected.json()
+        mldb.log(jsSelected)
+        result = jsSelected["output"]["score"]
+
+        self.assertAlmostEqual(result, 10)
+
+    def test_wine_quality_regression(self):
+        config = {
+            "type": "classifier.experiment",
+            "params": {
+                "trainingData": """
+                    select
+                    {pH, "fixed acidity"} as features,
+                    quality as label
+                    from wine_red
+                """,
+                "datasetFolds": [
+                        {
+                            "training_where": "rowHash() % 2 = 0", 
```

Comment:
It seems that there are still a couple of places where a configuration do not have the camel-case naming for fields.